README.md

```diff
@@ -26,14 +26,6 @@ OPTIONAL SECTIONS:
 After you're finished please remove all the comments and instructions!
 -->
 
-<div align="center">
-  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
-  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
-  <br/>
-
-
-</div>
-
 <!-- TABLE OF CONTENTS -->
 
 # 📗 Table of Contents
```

Comment:
- [ ] I am putting the review here because this is the only place I can put it in your README file, in your video presentation, you did not highlight  a piece of the code you finds interesting or is proud of making, kindly add that